src/components/shared/Button.js

```diff
@@ -1,8 +1,60 @@
-import styled from 'styled-components';
+import React from 'react';
 import _Button from '@material-ui/core/Button';
+import styled from 'styled-components';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Translate from '../Translate/Translate.react';
+import PropTypes from 'prop-types';
 
-const Button = styled(_Button)`
+const StyledButton = styled(_Button)`
   width: 10rem;
 `;
 
+const Button = props => {
+  const {
+    buttonText,
+    color,
+    variant,
+    isLoading,
+    isDisabled,
+    handleClick,
+    label,
+    style,
+  } = props;
+  return (
+    <StyledButton
+      variant={variant}
+      color={color}
+      onClick={handleClick}
+      disabled={isLoading || isDisabled}
+      label={label}
+      style={style}
```

Comment:
Pass `{...otherProps}` to the component here. This way we can ensure all props provided by MUI can be passed down to the button